### PR TITLE
[PR/Issue templates] normalize, group, sort + add myself for deepspeed

### PR DESCRIPTION
**Pull request by stas00**
This PR:
* case-normalizes, groups and sorts the tagging entries
* removes one duplicate
* adds myself for deepspeed
* adds/removes/moves others based on their suggestions through this PR

@LysandreJik, @sgugger, @patrickvonplaten

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -25,32 +25,23 @@ assignees: ''
...
- albert, bert, GPT2, XLM: @LysandreJik
- tokenizers: @mfuntowicz
- Trainer: @sgugger
- Speed and Memory Benchmarks: @patrickvonplaten
- Model Cards: @julien-c
- TextGeneration: @TevenLeScao
+ albert, bert, gpt2, xlm: @LysandreJik
+ bart, blenderbot, t5, speed and memory benchmarks, pegasus, longformer/reformer, marian, mbart: @patrickvonplaten
+ deepspeed, fsmt: @stas00
+ documentation, trainer: @sgugger
```
Comment (sgugger) [inline .github/ISSUE_TEMPLATE/bug-report.md:31]: I don't think it's helpful to mix different thins here: I liked having all models together before for instance, then other parts. So I'd rather reorganize by:
- models
- library components
- documentation
- examples

Comment (stas00) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: That works for me. Let's agree with others that this what everybody likes and I will adjust

Let me know if this works and if you want the cases to be different - they are inconsistent now.

Something like?

```
Models:
 albert, bert, xlm: @LysandreJik
 blenderbot, bart, marian, pegasus, encoderdecoder, longformer, reformer, t5: @patrickvonplaten
 fsmt: @stas00
 gpt2: @LysandreJik, @patrickvonplaten
 rag: @patrickvonplaten, @lhoestq
 tensorflow: @jplu
 transfoxl, xlnet: @patrickvonplaten, @TevenLeScao
 tokenizers: @mfuntowicz

Library:
 benchmarks: @patrickvonplaten
 deepspeed: @stas00
 text generationt: @patrickvonplaten, @TevenLeScao
 trainer, @sgugger

Documentation:
 documentation: @sgugger
 model Cards: @julien-c

HF projects: 
 rust tokenizers: [different repo](https://github.com/huggingface/tokenizers)
 nlp datasets: [different repo](https://github.com/huggingface/nlp)

Examples:
 examples/bert-loses-patience: @JetRunner
 examples/distillation: @VictorSanh
 examples/seq2seq: @patil-suraj
 examples/token-classification: @stefan-it
```

Comment (stas00) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: One more thing - the 2 sets (2 files) aren't the same. Should they be the same? And which one is the master then? I guess I can just merge the two inputs to get all the entry points.

Comment (sgugger) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: That's better! Now digging into the specifics I think (cc @LysandreJik and @patrickvonplaten ):
- under model you can add me for funnel
- for tokenizers we should remove poor Morgan. Do we put Anthony instead?
- The examples have changed. I think we should have a "maintained examples" line where you can put me and maybe @patil-suraj (if you don't mind starting helping maintaining all the new scripts?) and then two lines for the research projects mentioned (bert-loses-patience, distillation)

Comment (patil-suraj) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: I'm down for "maintained examples" @sgugger  :), also you could put me alongside Patrick for T5, bart, blender, marian, enc-dec

Comment (stas00) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: OK, let's sort out the PR template and then sync it with Issue template.

I applied the new structure - please make direct suggestions to the code for any desired changes. I applied most of your suggestions except Morgan/Anthony

Comment (patrickvonplaten) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: Let's also remove the model cards and I think we can also remove @TevenLeScao  from everything (or do you still want to be pinged @TevenLeScao ?)

Comment (sgugger) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: Just talked offline with @n1t0 and he is okay to replace Morgan for the tokenizers (Morgan is working on lots of other stuff at the moment, so doesn't need those notifications ;-) )
Agreed with the new structure @stas00 and all the changes suggested by @patrickvonplaten

Comment (stas00) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: pushed.

Comment (stas00) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: Once the PR template is complete and everybody is happy I will sync with the Issue template

Comment (TevenLeScao) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: I'm good @patrickvonplaten! I don't have time to follow, but I don't really mind being pinged, no big deal

Comment (stas00) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: oh, @TevenLeScao - I already removed you - revert?

Comment (TevenLeScao) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: It's fine, keep it that way, no worries

**Comment by stas00** [comment]
Once the PR template is complete and everybody is happy I will sync with the Issue template. So please only review the former if you're just joining in.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -37,26 +37,40 @@ members/contributors which may be interested in your PR.
...
+
+ examples/seq2seq: @patil-suraj, @sgugger
  examples/token-classification: @stefan-it
- documentation: @sgugger
- FSMT: @stas00
+
+Research Projects:
+
+ examples/research_projects/bert-loses-patience: @JetRunner
+ examples/research_projects/distillation: @VictorSanh
```
Comment (sgugger) [inline .github/PULL_REQUEST_TEMPLATE.md:74]: I was more thinking of:
```
Examples:

    maintained examples (not research project, not legacy) @sgugger @patil-suraj 
    research_projects/bert-loses-patience: @JetRunner
    research_projects/distillation: @VictorSanh
```
The new token classification example was written by me, so there is no need to tag stefan anymore on it.

**Review by sgugger — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (stas00) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done

**Comment by stas00** [comment]
should we add bullets?

```

Models:

- albert, bert, xlm: @LysandreJik
- blenderbot, bart, marian, pegasus, encoderdecoder, longformer, reformer, t5, transfoxl, xlnet: @patrickvonplaten
- fsmt: @stas00
- funnel: @sgugger
- gpt2: @patrickvonplaten, @LysandreJik
- rag: @patrickvonplaten, @lhoestq
- tensorflow: @jplu

Library:

- benchmarks: @patrickvonplaten
- deepspeed: @stas00
- ray/raytune: @richardliaw, @amogkam
- text generation: @patrickvonplaten
- tokenizers: @n1t0
- trainer: @sgugger

Documentation: @sgugger

HF projects:

- nlp datasets: [different repo](https://github.com/huggingface/nlp)
- rust tokenizers: [different repo](https://github.com/huggingface/tokenizers)

Examples:

- maintained examples (not research project or legacy): @sgugger, @patil-suraj
- research_projects/bert-loses-patience: @JetRunner
- research_projects/distillation: @VictorSanh
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -25,32 +25,23 @@ assignees: ''
...
  Please tag fewer than 3 people.
 
- albert, bert, GPT2, XLM: @LysandreJik
- tokenizers: @mfuntowicz
- Trainer: @sgugger
- Speed and Memory Benchmarks: @patrickvonplaten
- Model Cards: @julien-c
- TextGeneration: @TevenLeScao
+ albert, bert, gpt2, xlm: @LysandreJik
+ bart, blenderbot, t5, speed and memory benchmarks, pegasus, longformer/reformer, marian, mbart: @patrickvonplaten
```
Comment (patil-suraj) [inline .github/ISSUE_TEMPLATE/bug-report.md:29]:
```suggestion
 bart, blenderbot, t5, pegasus, marian, mbart: @patrickvonplaten, @patil-suraj 
 longformer, reformer, speed and memory benchmarks: @patrickvonplaten
```

Comment (stas00) [reply .github/ISSUE_TEMPLATE/bug-report.md]: oh, we are working on the PR template and then will sync the two, so let's focus on the former in this review. thanks.

I assume the two should be identical. Please correct me if I'm wrong.

Comment (patil-suraj) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Yes, the two should be identical IMO

**Comment by sgugger** [comment]
I like bullets.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -25,32 +25,38 @@ assignees: ''
...
- examples/seq2seq: @patil-suraj
- examples/bert-loses-patience: @JetRunner
- ray/raytune: @richardliaw @amogkam
- tensorflow: @jplu
- examples/token-classification: @stefan-it
- documentation: @sgugger
+Models:
+
+- albert, bert, xlm: @LysandreJik
+- blenderbot, bart, marian, pegasus, encoderdecoder, longformer, reformer, t5, transfoxl, xlnet: @patrickvonplaten
```
Comment (patil-suraj) [inline .github/ISSUE_TEMPLATE/bug-report.md:31]:
```suggestion
- blenderbot, bart, marian, pegasus, encoderdecoder,  t5: @patrickvonplaten, @patil-suraj
- longformer, reformer, transfoxl, xlnet: @patrickvonplaten
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -37,26 +37,36 @@ members/contributors which may be interested in your PR.
...
- examples/seq2seq: @patil-suraj
- examples/bert-loses-patience: @JetRunner
- tensorflow: @jplu
- examples/token-classification: @stefan-it
- documentation: @sgugger
- FSMT: @stas00
+Models:
+
+- albert, bert, xlm: @LysandreJik
+- blenderbot, bart, marian, pegasus, encoderdecoder, longformer, reformer, t5, transfoxl, xlnet: @patrickvonplaten
```
Comment (patil-suraj) [inline .github/PULL_REQUEST_TEMPLATE.md:43]:
```suggestion
- blenderbot, bart, marian, pegasus, encoderdecoder,  t5: @patrickvonplaten, @patil-suraj
- longformer, reformer, transfoxl, xlnet: @patrickvonplaten
```

**Review by patil-suraj — APPROVED** [review]
Bullets look good!

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 6]
```diff
@@ -25,32 +25,39 @@ assignees: ''
...
+- rag: @patrickvonplaten, @lhoestq
+- tensorflow: @jplu
+
+Library:
+
+- benchmarks: @patrickvonplaten
+- deepspeed: @stas00
+- ray/raytune: @richardliaw, @amogkam
+- text generation: @patrickvonplaten
+- tokenizers: @n1t0
```
Comment (LysandreJik) [inline .github/ISSUE_TEMPLATE/bug-report.md:45]: Could you add me to the tokenizers as well? Thanks.
```suggestion
- tokenizers: @n1t0, @LysandreJik
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 7]
```diff
@@ -25,32 +25,39 @@ assignees: ''
...
+- trainer: @sgugger
+
```
Comment (LysandreJik) [inline .github/ISSUE_TEMPLATE/bug-report.md:47]:
```suggestion
- pipelines: @LysandreJik 

```

**Review by LysandreJik — APPROVED** [review]
This looks great, thanks.

I've added myself to tokenizers and pipelines. @Narsil would you like to be pinged regarding pipelines as well, since you've been handling that part recently?

**Comment by stas00** [comment]
someone to tag for ONNX issues? @mfuntowicz?

**Pull request merged**